nominees/bemserver.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "BEMServer",
+  "description": "An open source Python server to deploy energy management solutions for buildings",
+  "website": "http://bemserver.org/",
+  "license": [
+    {
+      "spdx": "AGPL-3.0",
+      "licenseURL": "https://github.com/HIT2GAP-EU-PROJECT/bemserver/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 7,
+      "evidenceURL": "https://cordis.europa.eu/article/id/418007-middleware-system-utilises-ai-to-coordinate-building-energy-management"
+    },
+    {
+      "SDGNumber": 11,
+      "evidenceURL": "https://cordis.europa.eu/article/id/418007-middleware-system-utilises-ai-to-coordinate-building-energy-management"
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/HIT2GAP-EU-PROJECT/bemserver",
+  "organizations": [
+    {
+      "name": "HIT2GAP",
+      "website": "http://www.hit2gap.eu/",
```

Comment:
The website [link](http://www.hit2gap.eu/) appears to be broken. @rachita19082 kindly replace it with a valid one. Thanks.

Reply:
I have update the link in commit d39fd41.